docs/protos/home.md

```diff
@@ -10,7 +10,7 @@ slug: /
 
 Neosync has first-class APIs and SDKs that developers can use to integrate Neosync into their workflow. To learn more about how the Neosync API fits into the overall architecture, check out the Check out the [platform page](/platform).
 
-Neosync API serves up [Connect](https://github.com/connectrpc), which can listen using Connect, gRPC, or HTTP protocols. All of our APIs are generated from Protobuf files and our SDKs call gRPC endpoints. Alternatively, you can use our OpenAPI spec to generate REST clients as well.
+Neosync API serves up [Connect](https://github.com/connectrpc), which can listen using Connect, gRPC, or HTTP protocols. All of our APIs are generated from Protobuf files and our SDKs call Connect endpoints by default. Each SDK can be configured to call the gRPC or REST endpoints in Connect.
```

Comment:
Maybe: `Each SDK can be configured to use gRPC or REST in lieu of the default (Connect).`?